src/Storage/Services/RegisterService.cs

```diff
@@ -0,0 +1,115 @@
+        {
+            httpClient.BaseAddress = new Uri(platformSettings.Value.ApiRegisterEndpoint);
+            httpClient.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
+            _client = httpClient;
+            _httpContextAccessor = httpContextAccessor;
+            _generalSettings = generalSettings.Value;
+            _accessTokenGenerator = accessTokenGenerator;
+            _logger = logger;
+
+            _serializerOptions = new()
+            {
+                PropertyNameCaseInsensitive = true,
+                Converters = { new JsonStringEnumConverter() }
+            };
+        }
+
+        /// <inheritdoc/>
+        public async Task<Party> GetParty(int partyId)
+        {
+            Party party = null;
+
+            string endpointUrl = $"parties/{partyId}";
+            string token = JwtTokenUtil.GetTokenFromContext(_httpContextAccessor.HttpContext, _generalSettings.JwtCookieName);
+            string accessToken = _accessTokenGenerator.GenerateAccessToken("platform", "events");
+
+            HttpResponseMessage response = await _client.GetAsync(token, endpointUrl, accessToken);
+            HttpStatusCode responseHttpStatusCode = response.StatusCode;
+
+            if (responseHttpStatusCode == HttpStatusCode.OK)
+            {
+                party = await response.Content.ReadFromJsonAsync<Party>(_serializerOptions);
+            }
+            else
+            {
+                _logger.LogError("// Getting party with partyID {partyId} failed with statuscode {responseHttpStatusCode}", partyId, responseHttpStatusCode);
+            }
+
+            return party;
+        }
+
+        /// <inheritdoc/>
+        public async Task<int> PartyLookup(string orgNo, string person)
+        {
+            string endpointUrl = "parties/lookup";
+
+            PartyLookup partyLookup = new PartyLookup() { Ssn = person, OrgNo = orgNo };
+
+            string bearerToken = JwtTokenUtil.GetTokenFromContext(_httpContextAccessor.HttpContext, _generalSettings.JwtCookieName);
+            string accessToken = _accessTokenGenerator.GenerateAccessToken("platform", "storage");
+
+            StringContent content = new StringContent(JsonSerializer.Serialize(partyLookup));
+            content.Headers.ContentType = MediaTypeHeaderValue.Parse("application/json");
+
+            HttpResponseMessage response = await _client.PostAsync(bearerToken, endpointUrl, content, accessToken);
+            if (response.StatusCode == HttpStatusCode.OK)
+            {
+                Party party = await response.Content.ReadFromJsonAsync<Party>(_serializerOptions);
+                return party.PartyId;
+            }
+            else
+            {
+                string reason = await response.Content.ReadAsStringAsync();
+                _logger.LogError("// RegisterService // PartyLookup // Failed to lookup party in platform register. Response {response}. \n Reason {reason}.", response, reason);
```

Comment:
I don't want us to serialize the full response into an error message like this. Limit it to the response code. The body of the respons (reason) can also be pretty large, but let's keep it for now.

Reply:
What do you think of the warning below?

Reply:
Regarding the `PascalCase` warning, of course this need to be fixed, but it actually is like this in `altinn-events`, hereby it is kept this way to make the coding standard comply with other Altinn projects.